src/components/_templates/AddOrders/rules.ts

```diff
@@ -0,0 +1,8 @@
+export const orderFormRules = () => {
+  return {
+    rule: {
```

Comment:
Does it make sense to create common folder, add file with rules and put all common rules in this file ?